### ISSUE TITLE
stop using glog

### ISSUE THREAD

**Issue by miekg** [google/metallb#189]
**Is this a bug report or a feature request?**:

feature request

**What happened**:

metallb should probably stop using glog (unless it must, because being in google github org?). glog is a badly designed go package and we should stop it spreading even further.

See this bug report: https://github.com/coredns/coredns/issues/1597 and a hilarious twitter thread
* https://twitter.com/peterbourgon/status/972177627996831744

TL;DR: *anything* else is probably better suited.

**Comment by danderson** [comment]
LGTM, I have no special attachment to glog, I just used it because it's familiar from work.

Reference to a good structured logging library: https://godoc.org/github.com/go-kit/kit/log
